p2p/peer_tracker.go

```diff
@@ -72,23 +72,18 @@ func newPeerTracker(
 	}
 }
 
-// bootstrap will bootstrap the peerTracker with the given trusted peers and if
+// bootstrap will initiate connections to the given trusted peers and if
 // a pidstore was given, will also attempt to bootstrap the tracker with previously
 // seen peers.
 //
 // NOTE: bootstrap is intended to be used with an on-disk peerstore.Peerstore as
 // the peerTracker needs access to the previously-seen peers' AddrInfo on start.
 func (p *peerTracker) bootstrap(ctx context.Context, trusted []libpeer.ID) error {
-	// bootstrap connections to trusted
-	wg := sync.WaitGroup{}
-	wg.Add(len(trusted))
-	defer wg.Wait()
```

Comment:
Just trying to understand what happens here. Looks like bootsrap() is acting as an optional call, since it does not wait for any connections and does not even has retries. So there might be 0 connections in the end. Does bootsrap()  suppose to provide some connections to peer-tracker? If all of the connection fail would it result in some unwanted state? Would it make sense to add reconnects inside bootsrap in case 0 successful connections happened?